files/salt/openSUSE-15.0-x86_64_tiny/etc/ntp.conf

```diff
@@ -0,0 +1,154 @@
+
```

Comment:
This file provided within the salt/admin/etc/ntp.conf directory and should end up on the node anyways